### PR TITLE
66246 update golang cookbook to work for daptiv

### PR DESCRIPTION
**Pull request by ghost**
@heathsnow or @sneal  Sorry for repeat, I messed up my pull request and made it against wrong master.
 I need a tester to review my tests on #66246 and merge this pull request if everything looks good.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -15,4 +11,20 @@ rescue LoadError
...
-task :default => :foodcritic
+task :default => [:version, :tailor, :foodcritic, :spec]
+
+task :version do
+  IO.write('version.txt', (ENV['BUILD_NUMBER'] ? "0.0.#{ENV['BUILD_NUMBER']}" : '0.0.1'))
+end
+
+# FC043 is excluded because of the problem described in https://github.com/NOX73/chef-golang/issues/3
+FoodCritic::Rake::LintTask.new do |t|
+  t.options = { :fail_tags => ['any'], :tags => ['~FC043'] }
```
Comment (sneal) [inline Rakefile]: We shouldn't exclude FC043

**Comment by sneal** [comment]
Don't checkin the version.txt file and add it to the .gitignore

**Comment by ghost** [comment]
@sneal Let me know if my new commits are what you've requested.

**Comment by sneal** [comment]
:shipit:

**Pull request merged**